### ISSUE TITLE
Accessibility: Bring Search Submit in View on focus

### ISSUE THREAD

**Issue by davidakennedy** [WordPress/twentysixteen#279]
Originally brought up in #253:

In an ideal case, the search submit button should be brought into view on `:focus`. Because if a keyboard user focuses on it, they can get to it, but can't see it. This is not a concern for screen reader users, because they can hear it, which is what they need. So three options here to make this better for more users:
1. Just have the button not hidden off screen, and available to all users all the time. Users can still hit `enter` to search. This is probably the easiest and my favorite.
2. Bring the search button into focus when a user focuses on the form input. This means all users see the button and not just keyboard users.
3. Bring the search button into focus when a user focuses on the the button. Only keyboard users and screen reader users would see it, and benefit.

This is something that didn't happen with Twenty Fifteen, and paying a bit more attention to this would make it better for more users.

**Comment by iamtakashi** [comment]
> Just have the button not hidden off screen, and available to all users all the time. Users can still hit enter to search. This is probably the easiest and my favorite.

I can live with this idea as long as the button is a magnified glass icon and is inside of the input field. It wouldn't be an issue, would it?

**Comment by davidakennedy** [comment]
That is a good approach and would work really well.

**Comment by afercia** [comment]
I like this idea (n. 2) and maybe worth experimenting it also in the admin see https://core.trac.wordpress.org/ticket/31818

**Comment by davidakennedy** [comment]
I tested the PR, and it works perfectly, and harmonizes well with the other button styles. Markup looks good too. :+1:

**Comment by iamtakashi** [comment]
Thanks for testing. Let's go with this for this time.
